applications/HDF5Application/README.md

```diff
@@ -220,12 +227,24 @@ This process can be used to output data from model parts to HDF5. This will writ
                             "PRESSURE"
                         ]
                     },
+                    "element_integration_point_value_settings" : {
```

Comment:
just to say that this is inconsistent with the Core, there everything is called `GaussPoint...`

Think carefully if you want this, I can imagine this will cause problems in the future, e.g. when we finally manage to create a wrapper for all possible outputs

Reply:
done :)